src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -55,13 +55,14 @@ object AstTools {
       astNode match {
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlStringType => WdlString(t.getSourceString)
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString)
+        case t: Terminal if t.getTerminalStr == "identifier" && wdlType == WdlStringType => WdlString(t.getSourceString)
```

Comment:
Yes it felt weird but it wouldn't work without it because when you access an object member say myObject.myAttribute, myObject is a terminal `identifier` which was never getting resolved because there is no `case` for it.
Now convert it to a string may not be the right thing to do although it does seem to work.